src/features/commands.ts

```diff
@@ -40,10 +40,9 @@ export default function registerCommands(context: vscode.ExtensionContext, serve
     // running the command activates the extension, which is all we need for installation to kickoff
     disposable.add(vscode.commands.registerCommand('csharp.downloadDebugger', () => { }));
 
-    // register process picker for attach
-    let attachItemsProvider = DotNetAttachItemsProviderFactory.Get();
-    let attacher = new AttachPicker(attachItemsProvider);
-    disposable.add(vscode.commands.registerCommand('csharp.listProcess', async () => attacher.ShowAttachEntries()));
+    // register process picker for attach for legacy configurations.
+    disposable.add(vscode.commands.registerCommand('csharp.listProcess', async () => ""));
```

Comment:
Would we want to search for a special key instead of empty string?

Reply:
This seems to work pretty well to me. Though can we remove the `async` now? Or is this required to provide an async delegate?